nucypher/network/nodes.py

```diff
@@ -46,13 +46,13 @@
 from nucypher.utilities.logging import Logger
 
 TEACHER_NODES = {
-    TACoDomain.MAINNET.name: (
+    domains.MAINNET: (
```

Comment:
There are a few spots in the code that expect the key for the dictionary to be a string - eg. https://github.com/nucypher/nucypher/blob/b9e1a2705ac592cd423c815ce8c250865d0d9b8e/nucypher/characters/lawful.py#L1181.

Is that still valid?